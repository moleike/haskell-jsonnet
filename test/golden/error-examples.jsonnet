{
    assert self.f < self.g : 'wat',
    f: 1,
    g: 2,
}
