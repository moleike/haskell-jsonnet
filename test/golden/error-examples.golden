{
    "f": 1,
    "g": 2
}